src/components/modules/ui.js

```diff
@@ -352,6 +360,26 @@ export default class UI extends Module {
 
     }
 
+    clickedOnRedactorZone(event) {
```

Comment:
docs are missed

Reply:
And its duplicates https://github.com/codex-team/codex.editor/pull/242/files#diff-cbb94d819d5a907ec02909dbe8b598b1R238